composer.json

```diff
@@ -12,6 +12,7 @@
 		"ext-mbstring": "*",
 		"kint-php/kint": "^3.3",
 		"laminas/laminas-escaper": "^2.6",
+		"psr/http-message": "^1.0",
```

Comment:
With this PSR included in the main `require`, we should also include the library in `system/ThirdParty` for non-Composer users. Then have `ComposerScripts` updated to update this package as well,

Reply:
I injected the files and added them to the internal class map, but I don't see any `Psr\Log` handling in **ComposerScripts** so I didn't do anything there. If I missed something please let me know.